packages/web-embed-api/examples/web-components/lit/index.html

```diff
@@ -0,0 +1,8 @@
+<html>
+  <head>
+    <script src="./index.ts"></script>
```

Comment:
I tried it on my local and got an error, I think we need to add `type="modulo"` here, see the related [issue](https://github.com/fregante/browser-extension-template/issues/51) 
```suggestion
    <script type="module" src="./index.ts"></script>
```